app/src/main/java/edu/uc/zhukv/droneradarmap/MapsActivity.kt

```diff
@@ -79,7 +98,7 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
     private fun createTileProvider(): TileProvider {
         return object : UrlTileProvider(256, 256) {
             override fun getTileUrl(x: Int, y: Int, zoom: Int): URL {
-                val fUrl = String.format(OWM_TILE_URL, tileType, zoom, x, y)
+                val fUrl = String.format(tileURL, tileType, zoom, x, y)
```

Comment:
Again renamed to a more meaningful and simple name just to make it easier to understand.